src/resizing.jl

```diff
@@ -182,16 +191,43 @@ odims(original, i, short_size) = oftype(first(short_size), axes(original, i))
 
 """
     imresize(img, sz) -> imgr
-    imresize(img, inds) -> imgr
+    imresize(img, ratio::Real) -> imgr
+    imresize(img, inds::Integer...) -> imgr
+
+Change `img` to be of size `sz` (or to have indices `inds`). If `ratio` is used, then
+`sz = ceil(Int, size(img).*ratio)`. This interpolates the values at sub-pixel locations.
+If you are shrinking the image, you risk aliasing unless you low-pass filter `img` first.
+
+# Example:
```

Comment:
rather than having two `# Example:` sections, we should just have one `# Examples` section. (see https://docs.julialang.org/en/v1/manual/documentation/)